include/eve/module/real/algorithm/function/regular/simd/x86/all.hpp

```diff
@@ -1,51 +0,0 @@
-//==================================================================================================
-/**
-  EVE - Expressive Vector Engine
```

Comment:
why is it deleted ?

Reply:
It's dead and I know for a fact it's dead.

Reply:
Good